### PR TITLE
feat(AVO-3249): move relations queries to endpoint on proxy

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/AVO-3249

### CODE REVIEW

File: src/assignment/views/AssignmentOverview.tsx [hunk 1]
```diff
@@ -215,20 +207,6 @@ const AssignmentOverview: FunctionComponent<
...
-		if (query.sort_column) {
-			setSortColumn(query.sort_column as AssignmentOverviewTableColumns);
-		}
-		if (query.sort_order) {
-			setSortOrder(query.sort_order as Avo.Search.OrderDirection);
-		}
-		// Copy the query param value from the url once time when the page loads
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, []);
-
```
Comment (ian-emsens-shd) [inline src/assignment/views/AssignmentOverview.tsx:231]: `react-query` handles this now I assume?

File: src/admin/items/views/ItemDetail.tsx [hunk 2]
```diff
@@ -135,7 +136,10 @@ const ItemDetail: FunctionComponent<ItemDetailProps> = ({ history, match }) => {
 	};
 
 	const handleColumnClick = (columnId: string) => {
-		const sortDirection = queryParams.sortDirection === 'asc' ? 'desc' : 'asc'; // toggle
+		const sortDirection =
+			queryParams.sortDirection === OrderDirection.asc
+				? OrderDirection.desc
+				: OrderDirection.asc; // toggle
```
Comment (ian-emsens-shd) [inline src/admin/items/views/ItemDetail.tsx:142]: Replace with `toggleSortOrder` helper

File: tests/tests/beheer/t16-beheer-contentpagina-publiceren.spec.js
Comment (ian-emsens-shd) [inline tests/tests/beheer/t16-beheer-contentpagina-publiceren.spec.js:1]: Remove spec file

**Review by ian-emsens-shd — CHANGES_REQUESTED** [review]
See single comments

File: src/assignment/views/AssignmentOverview.tsx (hunk 1 above)
Comment (bertyhell) [reply src/assignment/views/AssignmentOverview.tsx:231]: no, we just don't save them locally in a useState anymore
but directly in the url, so we never need to sync between url and useStates

**Review by ian-emsens-shd — APPROVED** [review]

**Pull request merged**